src/olympia/search/filters.py

```diff
@@ -144,19 +145,29 @@ def get_value(self):
         # a specific API but rather encodes the guid and adds a prefix to it,
         # only in the search API): if the guid param matches this format, and
         # the feature is enabled through a setting, then we decode it and
-        # proceed. A 400 is returned if the format is recognized but the
-        # feature is disabled through the setting, acting as a kill-switch.
+        # check it against DiscoItems, which contains the list of add-ons
+        # susceptible to appear in disco pane, acting as a list of "safe"
```

Comment:
> susceptible to appear in disco pane

"selected to appear in disco pane"?

Reply:
Technically no. It's the list of items that we have an editorial translation for, which is used by TAAR as the list of items that *can* appear, but it doesn't mean they necessarily will. I'm open to better wording suggestions though.